README.md

```diff
@@ -0,0 +1,716 @@
+
+# 2. Delegation (Envelope)
+
+Delegations MUST include a signature that validates against the `iss` DID. A Delegation Payload on its own MUST NOT be considered a valid Delegation.
+
+| Field   | Type        | Required | Description                                  |
+|---------|-------------|----------|----------------------------------------------|
+| `pld`   | `&Payload`  | Yes      | The CID of the [Delegation Payload][Payload] |
+| `sig`   | `Signature` | Yes      | The `iss`'s [Signature] over the `pld` field |
+
+# 3. Delegation Payload
+
+The payload MUST describe the authorization claims, who is involved, and its validity period.
+
+| Field   | Type                              | Required | Description                                                 |
+|---------|-----------------------------------|----------|-------------------------------------------------------------|
+| `udv`   | `String`                          | Yes      | UCAN Semantic Version (`1.0.0-rc.1`)                        |
+| `iss`   | `DID`                             | Yes      | Issuer DID (sender)                                         |
+| `aud`   | `DID`                             | Yes      | Audience DID (receiver)                                     |
+| `nbf`   | `Integer` (53-bits[^js-num-size]) | No       | "Not before" UTC Unix Timestamp in seconds (valid from)     |
+| `exp`   | `Integer` (53-bits[^js-num-size]) | Yes      | Expiration UTC Unix Timestamp in seconds (valid until)      |
+| `nonce` | `Bytes`                           | Yes      | Nonce                                                       |
+| `meta`  | `{String : Any}`                  | No       | [Meta] (asserted, signed data) — is not delegated authority |
+| `sub`   | `DID`                             | Yes      | Principal that the chain is about (the [Subject])           |
+| `can`   | `String`                          | Yes      | The [Command] to eventually invoke                          |
+| `args`  | `{String : Any}`                  | Yes      | Any [Arguments] that MUST be present in the Invocation      |
+| `cond`  | `[Condition]`                     | Yes      | Any additional [Condition]s                                 |
+
+The payload MUST be serialized as [IPLD] and [signed over][Envelope]. The RECOMMENDED IPLD codec is [DAG-CBOR].
+
+## 3.1 Version
+
+The `udv` field sets the version of the UCAN Delegation specification used in the payload.
+
+## 3.2 Principals
+
+The `iss` and `aud` fields describe the token's principals. They are distinguished by having DIDs. These can be conceptualized as the sender and receiver of a postal letter. The token MUST be signed with the private key associated with the DID in the `iss` field. Implementations MUST include the [`did:key`] method, and MAY be augmented with [additional DID methods][DID].
```

Comment:
```suggestion
The `iss` and `aud` fields describe the token's principals. They are distinguished by having DIDs. These can be conceptualized as the sender and receiver of a postal letter. The token MUST be signed with a [verification method](https://www.w3.org/TR/did-core/#verification-methods) with [verification relationship](https://www.w3.org/TR/did-core/#verification-relationships) [`capabilityDelegation`](https://www.w3.org/TR/did-core/#capability-delegation) in the [DID document](https://www.w3.org/TR/did-core/#dfn-did-documents) that is the result of resolving the DID in the `iss` field of the UCAN. Implementations MUST include the [`did:key`] method, and MAY be augmented with [additional DID methods][DID].
```

Reply:
maybe 
`from the DID in the `iss` field of the UCAN. Implementations MUST include the [`did:key`] method, and MAY be augmented with [additional DID methods][DID].` 
-->
`from the DID in the `iss` field of the UCAN. Implementations MUST include the [`did:key`] method, and MAY be augmented with [additional DID methods][DID] that support the above flow.`

Reply:
> (Not all DID methods support the key names capabilityDelegation or arbitrary key names!)

I think it's fine with text as is. did-core defines capability delegation so that's all i need to refer to without worrying about methods that implemented it (fully/partially/correctly/incorrectly). If in practice someone is using UCANs with a did method that doesn't support capability delegations.... well IMO I think that should basically be inherently impossible but doesn't require spec text qualifications to point it out. did using method that doesn't support capability verification methods is just like using a did method that does, but the did doc has an empty array of them

Reply:
![image](https://github.com/ucan-wg/delegation/assets/37127325/ca58542c-42f9-4e74-affa-55ffcb5b6d3e)
very few of the registered DID methods implement this optional property, tho? IMHO everything that the DID method spec defines as optional is "rely on or assume support for at your own risk"...